### PR TITLE
feat(datepicker): add aria-* attrs and keyboard bindings to datepicker input

### PR DESCRIPTION
**Pull request by mmalerba**

### CODE REVIEW

File: src/lib/datepicker/datepicker-trigger.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import {ChangeDetectionStrategy, Component, Input, ViewEncapsulation} from '@angular/core';
+import {MdDatepicker} from './datepicker';
+
+
+@Component({
+  moduleId: module.id,
+  selector: 'md-datepicker-trigger, mat-datepicker-trigger',
```
Comment (kara) [inline src/lib/datepicker/datepicker-trigger.ts]: I thought we renamed this to `mdDatepickerToggle`?  Rebase error?

File: src/lib/input/input-container.ts [hunk 2]
```diff
@@ -381,4 +383,8 @@ export class MdInputContainer implements AfterContentInit {
 
     this._mdInputChild.ariaDescribedby = ids.join(' ');
   }
+
+  getPopupConnectionElementRef(): ElementRef {
```
Comment (kara) [inline src/lib/input/input-container.ts]: Didn't we decide to remove this?

File: src/lib/datepicker/datepicker.ts [hunk 3]
```diff
@@ -27,6 +27,9 @@ import {MdDatepickerInput} from './datepicker-input';
 import {CalendarLocale} from '../core/datetime/calendar-locale';
 
 
+let datepickerUid = 0;
```
Comment (kara) [inline src/lib/datepicker/datepicker.ts]: Add comment about what this is? We should also probably prefix with an underscore to be consistent with other counters.

**Comment by mmalerba** [comment]
comments addressed

**Pull request merged**